### DISCUSSION TITLE
Disable Achievements

### DISCUSSION THREAD

**Discussion by allo-** [community/community#32447] [Other Feature Feedback, Questions, & Ideas] [answered]
Is there a way to disable the achievements function?

I was fun to have a badge arctic vault contributor and maybe even the mars achievement. But now all kind of trivial things is added.
No, I don't need a badge for my first pull request. And I don't need a badge for having a repo with some stars either.

I would really appreciate an option to disable that section on the profile.

**Comment by xanderyzwich** [comment]
Or the ability to choose which achievements are visible.

**Reply by allo-** [reply]
This coupled with an option "add by default" or "disabled by default", so one doesn't have to hide new ones from time to time.

As said, some were funny, but now Github looks like it wants to be a game. You don't only get an achievement for a pull request, but you also unlock "bronze" and get a counter how many you opened, and finally it has a silly name like "Pull Shark".

I don't want to be the no-fun-allowed guy and publish most software for fun, but a code platform shouldn't be too playful, but a simple to use tool that makes it easy to focus on what you want to do.

**Reply by allo-** [reply]
Looks like there *is* a switch. Maybe you open another issue for selecting which ones to show.

**Comment by airtower-luna** [comment, marked as answer]
You can disable the achievements section on your profile, see here: https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/showing-your-private-contributions-and-achievements-on-your-profile#changing-the-visibility-of-achievements

**Comment by jfullstackdev** [comment]
latest update, you can now pick what you want to hide, unlike in the past, it was all or nothing

go to Achievements Tab, click the `Achievements` word, it is a link

![image](https://user-images.githubusercontent.com/47092464/204711684-e7c5ebb6-57f9-4fbf-b8fe-96c2182f41bb.png)

or this URL, it's same thing 

https://github.com/[username]?tab=achievements

you should be seeing like this 

![image](https://user-images.githubusercontent.com/47092464/204711801-dde96c4d-5133-4201-ad8a-9b23b55ed5d2.png)

then, click any badge, hover left then you should be seeing like this 

![image](https://user-images.githubusercontent.com/47092464/204711940-2c5e8b4d-7580-4145-afc0-268f1739d8de.png)

then boom ... you now have the option to hide those badges that you want to hide

**Comment by allo-** [comment]
What about an opt-in option?

I found the artic code contributor badge a nice thing that shows that my code got into the vault which was an interesting idea at that time.
But I think badges like "completed a pull request", "added a commit" and so on may be nice for people learning git, but are way to silly to want them on my profile. On the other hand, I also don't want to check after each action if I got a silly badge to hide.
So a third interesting option would be an opt-in mode. Hide all badges by default, but allow to select a few badges that should be shown.

**Comment by amesbury** [comment]
How about disabling it so I don't have to see this stuff in my own profile either?
